.gitignore

```diff
@@ -26,3 +26,5 @@ docs/code/api/*
 .idea
 .mypy_cache/
 .DS_Store
+
+venv
```

Comment:
Should add a line at the end of the file.
```suggestion
venv

```

Thinking of it again, maybe it is not a good idea to have this on the project's gitignore file — every developer can have a different name for its dev environment. 
I guess the best option is to add `venv` (or whatever is the name of your virtual environment) to your [global gitignore file](https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/).